src/api/getMovie.js

```diff
@@ -0,0 +1,12 @@
+import apiKey from "./apiKey";
+
+const getMovie = async (id, signal, plot = "short") => {
```

Comment:
Additionally it is advised that if function has more than two parameters it's better to use object which also helps to write self-documenting code.
```
getMovies = async({ id, signal, plot }...
```